presets/workspace/dependencies/requirements.txt

```diff
@@ -1,7 +1,7 @@
 # Dependencies for TFS
 
 # Core Dependencies
-vllm==0.9.0
+vllm==0.9.1
```

Comment:
We are in the process of upgrading vLLM to 0.10.1 (pending upstream release). Are you okay with holding off from merging this PR until 0.10.1 is ready?

Reply:
I think we could upgrade to vLLM 0.9.1 first, and then you could wait for 0.10.1 in your PR.

Reply:
It's ok. please ensure we upgrade to 0.10.1 when bump the service image tag.